db/structure.sql

```diff
@@ -1178,10 +1176,10 @@ CREATE INDEX index_metrics_on_tenant_id ON public.metrics USING btree (tenant_id
 
 
 --
--- Name: index_models_on_record_type_and_record_id; Type: INDEX; Schema: public; Owner: -
+-- Name: index_models_on_record; Type: INDEX; Schema: public; Owner: -
 --
 
-CREATE UNIQUE INDEX index_models_on_record_type_and_record_id ON public.models USING btree (record_type, record_id);
+CREATE UNIQUE INDEX index_models_on_record ON public.models USING btree (record_type, record_id);
```

Comment:
@HemaHG 

## Concern

Those SQL changes need to be documented  if no further work is done. My concern is that we would need to have migration path and documentation for existing customers to change this index name, and that is quite an unnecessary work for @3scale/documentation people and for customers while we can automatize it.


## The better solution is:

@HemaHG you can also make a migration that will change the name of this index. In the migration:

- delete the index `index_models_on_record_type_and_record_id` if it exists
- add the new index if it does not exist


The only documentation needed is for any upgrade the `zync-*` pods should be stopped. And it is already documented, so nothing to do in terms of doc. JFYI

Reply:
I will create a PR for migration